hw/ip/aes/data/aes.hjson

```diff
@@ -146,27 +153,27 @@
           Select encryption(0) or decryption(1) operation of AES unit.
         '''
       }
-      { bits: "3:1",
+      { bits: "4:1",
         name: "MODE",
-        resval: "1",
+        resval: "0x8",
         hwaccess: "hrw",
         desc:  '''
-          3-bit one-hot field to select AES block cipher mode: ECB (3'b001), CBC (3'b010) or CTR (3'b100).
-          Invalid input values, i.e., value with multiple bits set, and value 3'b000 are not supported are mapped to 3'b001.
+          4-bit one-hot field to select AES block cipher mode: ECB (4'b0001), CBC (4'b0010), CTR (4'b0100) or NONE (4'b1000).
```

Comment:
you may want to consider making these multibit fields for mode selection `enum` types. the reggen tool creates a nice table with a row for each value. see, e.g. https://github.com/lowRISC/opentitan/blob/5c323ef52ef42785286da49a3a2919a35dce75d9/hw/ip/pinmux/data/pinmux.hjson#L210-L226

Reply:
Good points, I switched to enums for both mode and key length fields.